Code/GraphMol/MolDraw2D/Qt/DrawTextFTQt.cpp

```diff
@@ -20,6 +20,8 @@ DrawTextFTQt::DrawTextFTQt(double max_fnt_sz, double min_fnt_sz,
                            const std::string &font_file, QPainter *qp)
     : DrawTextFT(max_fnt_sz, min_fnt_sz, font_file), d_qp(qp) {}
 
+DrawTextFTQt::~DrawTextFTQt() = default;
```

Comment:
Might as well put this directly in the header file

Reply:
That doesn't really work. If I put it into the header, I get the same error I mentioned above, probably because in the implementation file the header is imported before `QPainterPath`, so that if VS2019 generates the destructor as soon as it sees the `~DrawTextFTQt() = default`, it doesn't know how to destroy a `QPainterPath` yet. While if I put it into the implementation file, this happens when we have already imported `QPainterPath`, and know how to destruct it, so everything is fine.